app/graphql/mutations/create_therapist.rb

```diff
@@ -0,0 +1,30 @@
+class Mutations::CreateTherapist < Mutations::BaseMutation
+    argument :practice_id, Integer, required: true
+    argument :name, String, required: true
+    argument :address, String, required: true
+    argument :phone_number, String, required: true
```

Comment:
I know there was some talk to how phone number is being stored in terms of it's format. I know FE can do a check on the field to ensure it is of phone number status, but then how is it being sent back here? Are they cleaning out all the paraenthesis and dashes to ensure it's just a string of 10 numbers? This is also something the BE can ensure, BE can clean it to ensure all data in the DB is consistent: data integrity.

Same with address. Is address coming in as one long string with the city, state, and zip code? Sending everything in as different fields: street, city, state, zip, usually helps with database integrity ensuring formats are correct and information isn't missing. 

Just putting these thoughts out there for consideration.

Reply:
I see your point with the address. I think this will be a great edit for the next iteration/extension. I will bring this conversation the the FE for their consideration.